examples/timestamping/backend/src/api.rs

```diff
@@ -21,28 +21,39 @@ use schema::{Schema, TimestampEntry};
 use transactions::TxTimestamp;
 use TIMESTAMPING_SERVICE;
 
+/// The structure describes the query parameters for `handle_timestamp`
+/// and `handle_timestamp_proof` endpoints.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct TimestampQuery {
+    /// The hash of the given timestamp.
```

Comment:
`given` seems invalid here. We're not give any timestamp. And I'd prefer to not use `The` at the beginning of such lines. So `Hash of the requested timestamp`.